compiler/damlc/daml-stdlib-src/DA/Internal/Prelude.daml

```diff
@@ -380,7 +393,8 @@ infixr 1 =<<
 concatMap : (a -> [b]) -> [a] -> [b]
 concatMap f = concat . map f
 
--- | `replicate i x` is the list `[x, x, x, ..., x]` with `i` copies of `x`.
+-- | Create a list with the first argument number of copies of the second argument.
```

Comment:
I'm not sure this explanation adds value.

Reply:
fair enough, will remove